packages/editor/src/components/Editor.tsx

```diff
@@ -257,9 +261,8 @@ export const Editor: React.FC<Props> = observer(
                   display="flex"
                   flexDirection="column"
                   index={toolMenuTab}
-                  onChange={activatedTab => {
-                    setToolMenuTab(activatedTab);
-                  }}
+                  onChange={onRightTabChange}
+                  isLazy
```

Comment:
It can not be lazy, because there was an feature that everytime when user change tab, the scroll position should remain the last position. So the tab contents must be mounted all the time. Right? @xzdry

Reply:
Setting `isLazy` means that the tab panel's content will only be loaded when they are first activated, not all of them are mounted at the beginning
So there is no problem here, and I can subsequently control whether it keeps mounted when switching by setting `lazyBehavior`